pkg/client/interfaces.go

```diff
@@ -39,6 +44,15 @@ type LogConfig struct {
 	Out io.Writer
 }
 
+// Validate checks the config to determine if it is valid.
+func (lc *LogConfig) Validate() error {
+	if lc.Namespace == "" {
+		return errors.New("Namespace cannot be empty")
```

Comment:
nit: errors should be lower cased rather than using sentence capitalization. This applies throughout.
See: https://github.com/golang/go/wiki/CodeReviewComments#error-strings